OpenRA.Mods.Common/UpdateRules/Rules/20200503/UpdateTilesetColors.cs

```diff
@@ -0,0 +1,50 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	class UpdateTilesetColors : UpdateRule
+	{
+		public override string Name { get { return "Rename Tileset LeftColor and RightColor"; } }
+		public override string Description
+		{
+			get
+			{
+				return "The LeftColor and RightColor keys in isometric tilesets have been renamed to MinColor and MaxColor to reflect their proper usage.";
```

Comment:
Although maybe not intended you can also use them in rectangular maps. https://github.com/OpenHV/OpenHV/commit/da97b59f4eeaf4701cfe8fc492a7e14edd46075a

![image](https://user-images.githubusercontent.com/756669/85066507-6465ae80-b1af-11ea-82d2-fd64893109ac.png)

where they give an illusion of depth without me having to define yet another terrain type.